DEVELOPMENT.md

```diff
@@ -101,41 +101,41 @@
 
 ## Test Running
 
--   Run all unit tests.
+-   Run all unit and integration tests.
 
     ```sh
-    bazel test //...
+    bazel test ...
     ```
 
--   Run a single unit test like `JavaCodeGeneratorTest.java`
+-   Run all unit tests.
 
     ```sh
-    bazel run //src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest
+    bazel test $(bazel query ... | grep ^//:unit_)
     ```
 
--   Run a single integration test for API like `Redis`, it generates Java source
-    code using the Java microgenerator and compares them with the goldens files
-    in `test/integration/goldens/redis`.
+-   Run a single unit test like `JavaCodeGeneratorTest.java`
 
     ```sh
-    bazel test //test/integration:redis
+    bazel run :unit_com_google_api_generator_engine_JavaCodeGeneratorTest
```

Comment:
Is `bazel run` intentional here or a typo (instead of `bazel test`)? This does not seem right. Tests must be ran as `bazel test` if they are ran as `bazel run` it is not nonly non-idiomatic but also is not sandbozed by bazel has all sorts of problems coming with non-hermetic builds.